src/hotspot/share/runtime/suspendResumeManager.hpp

```diff
@@ -58,6 +58,7 @@ class SuspendResumeManager {
   bool suspend_with_handshake(bool register_vthread_SR);
 
   void set_suspended(bool to, bool register_vthread_SR);
+  void set_suspended_with_id(int64_t id, bool register_vthread_SR);
```

Comment:
Some descriptive comments would be useful now due to the difference in meaning for "suspended". Personally I think we should have `set_suspended` and `set_resumed` rather than passing true/false for the suspend state.

Reply:
I added comment. The set_suspended(bool..) -> set_suspended/set_resumed makes sense since we usually have have suspend/resume pair. I'll look on this.